test/integrations/destinations/klaviyo/processor/identifyTestData.ts

```diff
@@ -425,80 +428,6 @@ export const identifyData = [
   {
     id: 'klaviyo-identify-test-6',
     name: 'klaviyo',
-    description: 'Identify call without user custom Properties',
```

Comment:
Should we substitute with a positive scenario?

Reply:
Postive scenario is already covered in other test cases